core/css.js

```diff
@@ -386,9 +386,12 @@ Blockly.Css.CONTENT = [
   '.blocklyCommentTextarea {',
     'background-color: #ffc;',
     'border: 0;',
+    'outline: 0;',
     'margin: 0;',
     'padding: 2px;',
     'resize: none;',
+    'display: block;',
+    'overflow: none;',
```

Comment:
I think someone else has already got a PR for issue this is trying to fix (Chrome hides the textarea if the content of the text is too long that it needs to scroll), Chrome is basically broken here. The two options I see are: 
- do the overflow: none fix, which fixes this issue on Chrome, but then scrolling doesn't work on any browser, and a user would have to expand the bubble to see the full contents of the comment. 
- dont do the fix, and wait for Chrome to fix the issue.

Reply:
Yeah sorry, no such thing as overflow none. hidden is what I meant.